tools/qcxms/qcxms_getres.xml

```diff
@@ -0,0 +1,75 @@
+<tool id="qcxms_getres" name="QCxMS get results" version="@TOOL_VERSION@+galaxy0" profile="21.05">
+    <description>Get result of the Production run to obtain a QCxMS simulated mass spectrum</description>
+    
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="edam"/>
+    <expand macro="creator"/>
+    <expand macro="requirements"/>
+
+    <command detect_errors="exit_code"><![CDATA[
+        python3 '${get_res}' &&
+        /plotms_bin/PlotMS.v.6.2.0/plotms &&
+        sh ${__tool_directory__}/msp_out.sh
+    ]]></command>
+
+    <configfiles>
+        <configfile name="get_res">
+#set res_collection = str("', '").join([str($f) for $f in $res_files])
+
+with open('$mol', 'r') as file:
+    lines = file.readlines()
+    molname = lines[1]
+
+with open("molname.txt", "w") as file:
+    file.write(molname)
+
+input_files = '$res_collection'
+output_file = "tmpqcxms.res"
+
+with open(output_file, 'w') as output:
+    for file_path in input_files:
+        with open(file_path, 'r') as input_file:
+            output.write(input_file.read())
+
+        </configfile>
+    </configfiles>
+
+    <inputs>
+        <param type="data" name="mol" label="Molecule 3D structure [.xzy]" format="xyz,txt" />
+        <param type="data_collection" collection_type="list" name="res_files" label="res files [.res]" format="res,txt,text"/>
+    </inputs>
+
+    <outputs>
+        <data name="msp_output" format="msp" from_work_dir="simulated_spectra.msp" label="simulated_spectra.msp generated by ${tool.name} on ${on_string}"/>
+    </outputs>
+
+    <tests>
+        <test>
+            <param name="mol" value="mol.xyz" ftype="txt"/>
```

Comment:
```suggestion
            <param name="mol" value="mol.xyz" ftype="xyz"/>
```

Or does it not work otherwise? Maybe it would make sense to up the required galaxy profile to the version that knows the `xyz` datatype?

Reply:
Yes, the test fails. But I have bumped the profile version to 22.09 in cde84e0